tensorboard/webapp/widgets/custom_modal/custom_modal_component.ts

```diff
@@ -82,10 +82,11 @@ export class CustomModalComponent implements OnInit {
 
   public openAtPosition(position: {x: number; y: number}) {
     const root = this.viewRef.element.nativeElement;
-    const top = root.getBoundingClientRect().top;
-    if (top !== 0) {
-      root.style.top = top * -1 + root.offsetTop + 'px';
-    }
+    // Set left/top to viewport (0,0) if the element has another "containing block" ancestor.
```

Comment:
Have you tested this change internally? The page structure is slightly different in TB corp so it's worth checking that everything works as expected there.

Reply:
Just checked, the modals look good in 1p. IIUC this code should elegantly handle the [containing block](https://developer.mozilla.org/en-US/docs/Web/CSS/Containing_block#identifying_the_containing_block), however it is defined.

Reply:
That's great to hear, thanks for checking!